casr/src/bin/casr-afl.rs

```diff
@@ -130,15 +139,24 @@ fn main() -> Result<()> {
 
         // Get crashes from one node.
         let mut crash_info = casr::triage::CrashInfo {
-            casr_tool: casr_gdb.clone(),
             ..Default::default()
         };
-        crash_info.target_args = if matches.get_flag("ignore-cmdline") {
-            gdb_args.clone()
+        if matches.get_flag("ignore-cmdline") {
+            crash_info.casr_tool = tool_path.clone();
+            crash_info.target_args = args.clone()
         } else {
             let cmdline_path = path.join("cmdline");
             if let Ok(cmdline) = fs::read_to_string(&cmdline_path) {
-                cmdline.split_whitespace().map(|s| s.to_string()).collect()
+                let cmd_args: Vec<String> =
+                    cmdline.split_whitespace().map(|s| s.to_string()).collect();
+                if cmd_args[0].ends_with("dotnet") || cmd_args[0].ends_with("mono") {
+                    tool = "casr-csharp";
+                    crash_info.casr_tool = util::get_path("casr-csharp")?.clone()
```

Comment:
`clone` seems unnecessary, you forgot `;`

Reply:
done